actix-web/src/sec-5-errors.md

```diff
@@ -64,7 +65,7 @@ struct MyError {
    name: &'static str
 }
 
-/// Use default implementation for `error_response()` method
+// Use default implementation for `error_response()` method
```

Comment:
I think this makes a little more sense as a non-docblock comment, so removed one of the slashes.